src/jit/gentree.h

```diff
@@ -2199,6 +2198,92 @@ struct GenTree
     inline GenTree(genTreeOps oper, var_types type DEBUGARG(bool largeNode = false));
 };
 
+struct GenTreePhi : public GenTree
+{
+    struct Use
+    {
+        GenTree* op;
```

Comment:
I don't think we have a policy, guideline or even consensus around this, but I find node to be more meaningful than 'op', which at least in the JIT is sometimes used to mean an operator and sometimes an operand (and yes, we've got `op1` and `op2` all over the place, but still ...)

Reply:
`node` sounds pretty good to me too, I'll update. Anyway, "use" and "operand" seem to overlap a bit.